<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaginaPrincipal</class>
 <widget class="QMainWindow" name="PaginaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>491</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>90</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(186, 0, 0);
background-color: rgb(51, 51, 51);</string>
    </property>
    <property name="text">
     <string>RentCar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI Light</family>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);
color: rgb(236, 236, 236);</string>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_busqueda">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 3px;
border-color: rgb(186, 0, 0);
background-color: rgb(51, 51, 51);
color: rgb(186, 0, 0);
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>Sistema de busqueda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_salida">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Tai Le</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
border-width: 3px;
border-color: rgb(186, 0, 0);
background-color: rgb(51, 51, 51);
color: rgb(186, 0, 0);
border-radius: 25px;</string>
    </property>
    <property name="text">
     <string>Programacion de salidas</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
